mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+import {ReactElement} from "react";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed values
+   */
+  value: string | number;
+
+  /**
+   * Displayed placeholder
+   */
+  placeholder: string;
+
+  /**
+   * Function to render value
+   */
+  renderValue: (value: string | number) => ReactElement;
+
+  /**
+   * Function to render placeholder
+   */
+  renderPlaceholder: (placeholder: string) => ReactElement;
+}
+
+/**
+ * Component Text
+ */
+export const Text = (props: TextProps) => {
+  const {
+    placeholder,
+    value,
+    renderPlaceholder,
+    renderValue,
+  } = props;
```

Comment:
Delete this/ We don't use destructure for props in the project

Reply:
deleted destructure